README.md

```diff
@@ -21,7 +21,7 @@ Consider this file name:
 What is the purpose of this file, and what is the purpose of the numbers at the beginning of its name?
 
 ```text
-Your answer...
+This is a migration file.  Migrations are used to import data that create tables and classes and define columns in the database schema.  The above file name begins with a timestamp and includes a description with underscores on what action it is undertaking.
 ```
```

Comment:
:+1: Also important to note that the timestamp indicates when a migration should be run relative to others.